portal/models/questionnaire_bank.py

```diff
@@ -261,7 +261,7 @@ def validate_classification_count(qbs):
             if qbs and qbs[0].classification == 'recurring':
                 return
             if len(qbs) > 1:
-                current_app.logger.error(
+                current_app.logger.warn(
```

Comment:
Would it be easy to condition this on `ROLE.TEST`? 

I feel like these would be valuable warnings for real patients, and would "encourage" us to properly flag out test patients as such :)

Reply:
For one, in our non-prod instances, this is true for LOTS of users. Unless we want to flag almost all users in non-prod instances with `ROLE.TEST`, we're going to keep getting a _lot_ of noise from all those instances. For instance, with stg-eproms alone, one run of the site summary email process for just one of the top orgs generated 38 emails for this error, and it would have been more if it hadn't errored out (because of the `user.email` issue)

For two, with prod instances, how likely is it that we'll see this error, and proactively fix it (esp for real patients)? Seems more likely something to help troubleshoot, if a user's asking why the wrong QB is showing up in their UI.

Reply:
(per Slack convo, going to keep this as `logger.error`, but only log in production)